meetings/summaries/2025.md

```diff
@@ -0,0 +1,16 @@
+## 2025 Technical Committee Summary
+
+
+2025 has been a decisive year for the Technical Committee (TC), marked by strengthened governance, a more mature incubation framework, and steady growth in the OpenRail project ecosystem.
+
+The election of **Florian** as chair of the technical committee, **first representative of a project** demonstrating the maturity of the technical committee's governance.
+
+The TC completed and formalized both Stage 2 and Stage 3 incubation criteria, enabling more consistent project evaluation. Incubation stages were renamed to **Onboarded → Qualified → Adopted**, providing clarity for members, contributors, and external partners. We published a breakdown and description of the capabilities of the railway system to help openrail projects and future users to meet up.
+
+On the project side, several initiatives advanced significantly. **trainpositioning-lib** underwent licensing and technical review; **PopUpSim** was fully approved and integrated in **DAC migration**; **AMS** continued progressing toward a complete proposal; and **Netzgraphik-Editor** (NGE) successfully migrated into the ORA ecosystem. The TC also analyzed multiple Hack4Rail 2025 projects, identifying promising candidates for future incubation. Core projects such as **OSRD**, **NGE**, **RCM-dx** and **LibLRS** also made notable improvements, including new features, documentation, bug fixes.
+
+Infrastructure and tooling remained a strategic priority. The TC began planning an OpenRail Matrix server, refined hosting policies for **demo environments**, and initiated updates to security templates in anticipation of the Cyber Resilience Act.
```

Comment:
We did not only plan a Matrix server but actually run it today, starting as support for Hack4Rail. Same with Nextcloud and a proper user management.

Regarding demo, I would find the OSRD demo instance worthy to mention: a success for OSRD, but also for the TC that faciliated the process.